Source/WebCore/dom/StaticRange.cpp

```diff
@@ -83,4 +83,14 @@ void StaticRange::visitNodesConcurrently(JSC::AbstractSlotVisitor& visitor) cons
     addWebCoreOpaqueRoot(visitor, end.container.get());
 }
 
+bool StaticRange::isValid() const
+{
+    unsigned containerLength = startContainer().length() + endContainer().length();
```

Comment:
This may overflow and we should probably return false if it does, no?

See `CheckedUint32` type to deal with overflows.